insights/parsers/bond.py

```diff
@@ -83,6 +86,7 @@ class Bond(Parser):
     def parse_content(self, content):
         self._bond_mode = None
         self._partner_mac_address = None
+        self.xmit_hash_policy = None
```

Comment:
Would you please add the new attribute to docstring?

Reply:
added attribute string